decidim-initiatives/config/locales/en.yml

```diff
@@ -264,6 +264,8 @@ en:
             update: Update
           form:
             authorizations: Authorization settings
+            child_scope_threshold_enabled_help: This config flag doesn't support offline votes, it enables sub-jauges and works with an authorization handler that associate associates a scope to the user, make sure you select that authorization bellow in authorization settings. For more info on how this configuration works see this <a href="https://docs.decidim.org/admin-manual/en/initiatives/" target="_blank">link</a>. This is usually activated along together with "Only allow global scope initiative creation".
```

Comment:
This config flag doesn't support offline votes, it enables sub-jauges and works with an authorization handler that associates a scope to the user, make sure you select that authorization, bellow in authorization settings. For it to work scopes need to be configured in hierarchical way : 1 Parent - N Child. For more info on how this configuration works, see this <a href="https://docs.decidim.org/admin-manual/en/initiatives/" target="_blank">link</a>..